test/LibraryManager.IntegrationTest/LibmanCompletionTests.cs

```diff
@@ -33,5 +32,152 @@ public void LibmanCompletion_Destination()
 
             LibmanTestsUtility.WaitForCompletionEntries(Editor, expectedCompletionEntries, caseInsensitive: true);
         }
+
+        [TestMethod]
+        public void LibmanCompletion_Provider()
+        {
+            _libManConfig.Open();
+
+            string[] expectedCompletionEntries = new[] {
+                "cdnjs",
+                "filesystem",
+                "unpkg",
+            };
+
+            Editor.Caret.MoveToBeginningOfFile();
+            Editor.Caret.MoveDown(4);
+            Editor.KeyboardCommands.Type("\"provider\":");
+
+            LibmanTestsUtility.WaitForCompletionEntries(Editor, expectedCompletionEntries, caseInsensitive: true);
+        }
+
+        [TestMethod]
+        public void LibmanCompletion_DefaultProvider()
+        {
+            _libManConfig.Open();
+
+            string[] expectedCompletionEntries = new[] {
+                "cdnjs",
+                "filesystem",
+                "unpkg",
+            };
+
+            Editor.Caret.MoveToBeginningOfFile();
+            Editor.Caret.MoveDown();
```

Comment:
Same as above, use a regex to place the caret, then it enter.  E.g. 

    Editor.Caret.MoveToExpression(@"""version"": ""1.0"",);
    Editor.Caret.MoveToEndOfLine();
    Editor.KeyboardCommands.Enter();

Reply:
Will do

---
In reply to: [193924933](https://github.com/aspnet/LibraryManager/pull/108#discussion_r193924933) [](ancestors = 193924933)